src/custom/pages/Faq/index.tsx

```diff
@@ -173,6 +173,15 @@ export default function Faq() {
             features while placing gas free trades.
           </p>
 
+          <h3 id="why-is-cowswap-a-meta-dex-aggregator">Why is CowSwap a Meta DEX aggregator?</h3>
```

Comment:
I might reword this to something like
```suggestion
          <h3 id="why-is-cowswap-a-meta-dex-aggregator">What makes CowSwap a "Meta" DEX aggregator?</h3>
```